src/test/java/net/snowflake/client/util/StopwatchTest.java

```diff
@@ -26,17 +26,16 @@ public void before() {
   @Test
   public void testGetMillisWhenStopped() throws InterruptedException {
```

Comment:
let's not change Stopwatch tests if not necessary

Reply:
I reverted iit back.